aiographite/aiographitesend.py

```diff
@@ -0,0 +1,325 @@
+import asyncio
+from .graphite_escaping import GraphiteEncoder
+import pickle
+import struct
+import socket
+import os
+import time
+from typing import Dict, Tuple, List
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class PlaintextProtocol(object):
```

Comment:
this file is still a bit long: ~ 300 lines. We should be shooting for ~ 100 lines, if possible.

We should move the protocols to a different module. That helps keep small context on your code.
```
from .protocol import PlaintextProtocol, PickleProtocol
# or
from .protocol.plaintext import PlaintextProtocol
from .protocol.pickle import PickleProtocol
```